### DISCUSSION TITLE
Toa lightning fixer possible?

### DISCUSSION THREAD

**Discussion by Justin13467** [runelite/runelite#16958] [Ideas]
This is my first time with an idea, and im doing toa right now going for my 500. I was hoping to find a plugin that removed  the remainder of an animation of an attack that has already completed. Im talking specifically in regards to the lightning attacks on final phase wardens. Is there any way to implement a simple plugin that removes completed attack animations to allow players to see tiles that are actually safe? I only ask because when the lightning part of the animation goes off, the tile is already safe to stand on and if it possible to remove that last remaining tick of animation then it would be easier for all players to understand the mechanics. This plugin (if made by someone) would work similar to the remove dead npcs plugin feature of entity hider, but called something like remove completed attack animations. 

And if thats not possibly, maybe have a plugin that speeds up the animation by 1 tick so the lightning part of the animation actually triggers on the damage being dealt portion of the attack. Both of these would be workable solutions if allowed.

**Comment by Nightfirecat** [comment]
I'm inclined to say this isn't allowed based on the following line from [Jagex's Third Party Client Guidelines](https://secure.runescape.com/m=news/third-party-client-guidelines?oldschool=1): (emphasis mine)
> Anything that automatically indicates where to stand, **or not to stand**. This applies to only automatic indicators, and not tiles which have been manually marked.

**Reply by Justin13467** [reply]
In my opinion, if remove dead npcs and other animation hiding tools in that plugin are allowed then this should be as well. it would not be telling someone where to stand, it essentially would be removing clutter of finishing animations on the screen, which then the player can make a better choice of what to do with more accurate information.but they wouldn’t be being shown where to click, since the animation will already be ending. this would just be removing projectile animations early(which entity hider already can hide projectiles if you want to classify lightning as such) or having the animation of lightning striking actually triggering with the damage. It seems very confusing as to why the animation was developed liked that by jagex because as of right now, you move to the tile right before the lightning part of the animation goes off, and then get struck by the lightning and take no damage because the damage was already done before the lightning part of the animation.

**Reply by Nightfirecat** [reply]
Apologies, I guess highlighting **where to stand** would have been a better indicator. As for dead NPC hiding, that is something explicitly allowed and endorsed by Jagex, where this seems to be the opposite.
